azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -736,6 +741,11 @@ private void createPod(final int executionId) throws ExecutorManagerException {
     flow.setStatus(Status.PREPARING);
     flow.setVersionSet(versionSet);
     this.executorLoader.updateExecutableFlow(flow);
+    // Record time taken to dispatch flow to a container
+    if (flow.getSubmitTime()>0) {
+      final long containerDispatchDuration = System.currentTimeMillis() - flow.getSubmitTime();
```

Comment:
Is it possible to convert this time into seconds?

Reply:
I hope to keep consistency as all the other time metrics uses milliseconds.